server/controllers/finance/cash.js

```diff
@@ -249,15 +267,41 @@ exports.update = function update(req, res, next) {
   })
   .catch(next)
   .done();
-};
+}
 
 /**
  * DELETE /cash/:uuid
  * Reverses a cash payment using the cash discard table
  * @TODO - should this be implemented as a separate API?
  */
-exports.debitNote = function debitNote(req, res, next) {
+function debitNote(req, res, next) {
   'use strict';
   // TODO
   next();
-};
+}
+
+
+/**
+ * retrieves cash payment uuids from a reference string (e.g. HBB123)
+ * GET /cash/references/:reference
```

Comment:
This is `jsdoc` syntax.  The `@description` is optional if the first line is the description ([src](http://usejsdoc.org/tags-description.html)).  Can you be more specific about what you'd like to see?